src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,17 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        OptionalInt min = numbers.stream()
+                .map(x -> Arrays.stream(x.split(","))
+                        .collect(Collectors.toList()))
+                .flatMap(Collection::stream)
```

Comment:
```suggestion
                .map(s -> s.split(","))
                .flatMap(Arrays::stream)
```
It can be simplified like this)